pysph/base/octree.pyx

```diff
@@ -456,7 +683,7 @@ cdef class Octree:
 
         """
         cdef NNPSParticleArrayWrapper pa_wrapper = NNPSParticleArrayWrapper(pa)
-        return self.c_build_tree(pa_wrapper)
+        return self.c_build_tree(pa_wrapper, threshold = 40000)
```

Comment:
This should be `threshold` and not `threshold=40000` otherwise setting the threshold has no effect.